app/models/problem.rb

```diff
@@ -0,0 +1,7 @@
+class Problem < ActiveRecord::Base
+  validates_presence_of :title
+  validates_presence_of :description
+  validates_presence_of :status
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.